app/assets/javascripts/component_guide/filter-components.js

```diff
@@ -0,0 +1,36 @@
+(function() {
+  window.GOVUK = window.GOVUK || {};
+
+  window.GOVUK.FilterComponents = function filterList(searchTerm) {
+    var itemsToFilter = document.querySelectorAll('.component-list li');
+
+    // Scroll so results are more visible
+    var position = document.querySelector('[data-module="filter-components"]').offsetTop;
+    window.scrollTo({top: position});
```

Comment:
While I see the intent this results in a bit of a weird case where, while you type (e.g. `Details`) the input moves constantly around the page making it harder to follow. It may be only me – or we can get a user researcher involved and see how others feel about it 😁.